_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -37,21 +38,17 @@ Each test target's `autocomplete` [attribute value][] is a [space separated][] l
 3. An optional token of either "home", "work", "mobile", "fax" or "pager", only if the last token is "email", "impp", "tel" or "tel-\*"; then
 4. A required token from the [correct autocomplete field][].
 
-## Expectation 2
```

Comment:
You still have some failing examples that now pass the rule. Can you take those out and add in a passed example showing this should no longer be failed?

Reply:
Will add an example with an incorrect type. Which failed example is now passing? I've looked through them again and I'm not sure which ones you mean. 
I've also checked the examples with optional tokens and if they are in the wrong order, querying the `autocomplete` returns an empty string, which is in line with what we currently have.